portality/bll/services/article.py

```diff
@@ -184,15 +198,17 @@ def is_legitimate_owner(self, article, owner):
         if len(owners) > 1:
             return False
 
+        # if the found owner is not the same as the desired owner, return false
+        if owners[0] != owner:
+            return False
+
         # single owner must still know of all supplied issns
-        compare = list(set(seen_issns[owners[0]]))
-        if len(compare) == 2:   # we only want to check issn parity for journals where there is more than one issn available.
-            for issn in issns:
-                if issn not in compare:
-                    return False
-
-        # true if the found owner is the same as the desired owner, otherwise false
-        return owners[0] == owner
+        journal_issns = set(seen_journal_issns[owners[0]])
```

Comment:
we've already established that `owners[0] == owner`, so they're equivalent. I'd prefer to use the singular `owner` here, but it doesn't really matter.